test/onemkl.jl

```diff
@@ -0,0 +1,16 @@
+using oneAPI
+using oneAPI.oneMKL
+using LinearAlgebra
+
+m = 20
+n = 35
+k = 13
+
+########################
+@testset "level 1" begin
+	@testset for T in eltypes
+		if T <:oneMKL.onemklFloat
+			@test testf(rmul!, rand(T,m), rand())
```

Comment:
See the comment above on the dispatching rules.  The rand() function on the 'alpha' only creates 'float' type numbers so not all combinations of 'alpha' - ComplexF32, ComplexF64 are tested in this test.

Reply:
Yes agree, I have reenabled the original rand(T,1) to launch float, complex type tests. The cross combination ComplexF32 [x] vs Float32 [Alpha] scenarios I will write separate tests.